### DISCUSSION TITLE
MQTT Telegramme senden

### DISCUSSION THREAD

**Discussion by hghgsat** [brutella/hkknx-public#221] [Ideas]
Hi,
kann man via Script MQTT Telegramme senden? Ich würde gerne von meiner Wetterstation und auch anderen Vorgängen am KNX-Bus per MQTT ein Log füttern. Es stellt sich dann auch die Frage, ob man generell ein Event benötigt, um auf quasi "alle" Telegramme zu reagieren bzw. diese zu protokollieren, ohne das man gleich zig versch. Scripts braucht.
thx
Harry

**Comment by thorsten-gehrig** [comment]
Eine MQTT integration wäre super...

**Comment by brutella** [comment]
Danke für das Feedback. MQTT Integration ist definitiv interessant.
